src/main/java/com/coveros/selenified/element/Get.java

```diff
@@ -418,8 +410,7 @@ public int numOfTableColumns() {
     public List<WebElement> tableRow(int rowNum) {
         List<WebElement> rows = tableRows();
         if (rows == null) {
-            return null; // NOSONAR - returning an empty array could be confused
-            // with a row out of range
+            return null; // returning an empty array could be confused with a row out of range
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1168)